packages/eslint-config-eslint/base.js

```diff
@@ -255,7 +255,6 @@ const unicornConfigs = [{
         "unicorn/prefer-at": "error",
         "unicorn/prefer-includes": "error",
         "unicorn/prefer-set-has": "error",
-        "unicorn/prefer-string-slice": "error",
```

Comment:
`String.prototype.slice` is already enforced by `no-restricted-properties`: https://github.com/eslint/eslint/blob/ee5619ee8f7cf9089557d8e133936608ae5584a0/packages/eslint-config-eslint/base.js#L62-L65

I removed this line to avoid a double report when `.substring` is used.

Reply:
Maybe it would be better to keep the unicorn rule because it's autofixable and instead remove the `no-restricted-properties` config entry for `"substring"` property?

Reply:
Good idea.